utils/search/backends/solr555pysolr.py

```diff
@@ -246,6 +266,10 @@ def convert_sound_to_search_engine_document(self, sound, include_analyzer_output
                         suffix = SOLR_DYNAMIC_FIELDS_SUFFIX_MAP.get(type(value), None)
                         if suffix:
                             document[f'{key}{suffix}'] = value
+                            if suffix == '_ls':
+                                # For dynamic fields of type "list of strings", we also need to set an extra field that
+                                # will be used for faceting
+                                document[f'{key}{suffix}{SOLR_DYNAMIC_FIELD_FACET_EXTRA_SUFFIX}'] = value
```

Comment:
instead of "special casing" this, we could instead make `SOLR_DYNAMIC_FIELDS_SUFFIX_MAP` be a map of type -> list of suffixes. So for the case of list we just have the _ls and _ls_f suffixes.

Reply:
Thanks for the suggestion. I think this would complicate the logic in other parts of the code where `SOLR_DYNAMIC_FIELDS_SUFFIX_MAP`. I think the way it is now is not that bad with the comment.